### ISSUE TITLE
strange error when travel graph

### ISSUE THREAD

**Issue by Himan000** [tariqdaouda/pyArango#130]
FOR v IN OUTBOUND SHORTEST_PATH 'airports/BIS' TO 'airports/JFK' flights RETURN v
when run above query，an error occur as below：
Query: AQL: collection not known to traversal: 'airports'. please add 'WITH airports' as the first line in your AQL (while executing)
It's strange. how to solve it? thanks

**Comment by ifcologne** [comment]
WITH airports FOR v IN OUTBOUND SHORTEST_PATH 'airports/BIS' TO 'airports/JFK' flights RETURN v

**Comment by Himan000** [comment]
Yes, it's ArangoDB cluster. So how can I do to make the AQL runable, this is a official example.
